### PR TITLE
cart and wishlist page

### PR DESCRIPTION
**Pull request by praveenreddy1798**
**Wishlist Page**

Screenshot 2022-02-18 at 2 08 56 AM

**Cart Page**

Screenshot 2022-02-18 at 2 09 12 AM

### CODE REVIEW

File: containers/cart-page/index.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.quantity-section > * {
+  background-color: transparent;
+}
+
+.increase,
+.decrease {
```
Comment (vaibhav18matere) [inline containers/cart-page/index.css]: use more descriptive names like `increase-items` and `decrease-items`

File: containers/cart-page/index.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  height: 2rem;
+  border: 1.5px solid var(--secondary);
+  text-align: center;
+  font-size: 1.5rem;
+}
+
+.quantity {
+  width: 3rem;
+  height: 2rem;
+  border: 1.5px solid var(--secondary);
```
Comment (vaibhav18matere) [inline containers/cart-page/index.css]: border is having similar properties,  you can make it to separate utility class and use in cart directly

**Review by vaibhav18matere — APPROVED** [review]

**Review by manojasarsa — APPROVED** [review]
good to go!

**Review by TejasShekar — APPROVED** [review]
Looks good to merge.

File: containers/cart-page/index.css (hunk 1 above)
Comment (praveenreddy1798) [reply containers/cart-page/index.css]: yes, will change this

File: containers/cart-page/index.css (hunk 2 above)
Comment (praveenreddy1798) [reply containers/cart-page/index.css]: yes, make sense

**Pull request merged**